packages/core/src/util/normalizeConfig.js

```diff
@@ -674,3 +683,27 @@ function normalizeSingleValue(configValue, defaultValue, configPath, envVarKey)
   }
   return configValue;
 }
+/**
+ * @param {InstanaConfig} config
+ */
+function normalizeIgnoreEndpoints(config) {
+  if (!config.tracing?.ignoreEndpoints) {
```

Comment:
```suggestion
  if (!config.tracing.ignoreEndpoints) {
```
Already handled in normalizeTracingConfig